src/main.cpp

```diff
@@ -4318,7 +4330,7 @@ CBlockIndex * InsertBlockIndex(uint256 hash)
     return pindexNew;
 }
 
-bool static LoadBlockIndexDB()
+bool static LoadBlockIndexDB(bool* fRebuildRequired)
```

Comment:
Out of curiosity, why `bool *`? Is there some third state this function has to handle outside of `true` `false`?